### ISSUE TITLE
False positive for table-groups

### ISSUE THREAD

**Issue by lstrzebinczyk** [ember-template-lint/ember-template-lint#784]
Hello :wave: 

This piece of template:

```
<table>
  <thead></thead>
  {{#if this.thing}}
    <tfoot></tfoot>
  {{/if}}
</table>
```

Triggers the table groups rule when used via ember-cli-template-lint, and returns me an error:

```
web/templates/components/federation-site/table.hbs
  1:0  error  Tables must have a table group (thead, tbody or tfoot).  table-groups
```

However, when I try to add a failing test case here, it accepts it and finds no error. 

This happens when I use

```
    "ember-cli-template-lint": "^1.0.0-beta.3",
    "ember-template-lint": "^1.4.0",
```

Which I believe to be most recent versions.

**Comment by bmish** [comment]
I tried the template you provided and there was no lint violation reported for me. You may need to pin `ember-template-lint` to the latest version with a yarn resolution, since `ember-cli-template-lint` is still specifying an older version of `ember-template-lint`.

**Comment by lstrzebinczyk** [comment]
Ha, I removed `yarn.lock` and forced it to rebuild. When I just `yarn`ed yesterday it had both versions `1.1.0 ` and `1.4.0` in lock, but when it rebuilt completely it settled on `1.4.0` and the problem is solved.

So I guess I'm sorry to bother, as this one was on me; ). Thank you for your time, closing!
